DDDEastAnglia.Tests/Controllers/SessionControllerTests.cs

```diff
@@ -73,29 +69,63 @@ public void CannotBeginToDeleteASession_WhenTheSessionBelongsToAnotherUser()
         public void CannotConfirmToDeleteASession_WhenTheSessionBelongsToAnotherUser()
         {
             var session = new Session { SessionId = 1, SpeakerUserName = "bob" };
-            var controller = SessionControllerFactory.Create(session);
+            var controller = new SessionControllerBuilder().Updating(session).Build();
 
             var result = controller.DeleteConfirmed("fred", 1);
 
             Assert.That(result, Is.InstanceOf<HttpUnauthorizedResult>());
         }
 
-        private static class SessionControllerFactory
+        [TestFixture]
+        public class EmailTests
```

Comment:
I know what you mean. I did it for explicit categorisation - these tests cover a subset of the behaviour of the `SessionController`. I then felt compelled to follow the conventions in the file, so ended up with the clunky names. Perhaps I'll remove the nested class and just make the two tests top-level tests.